CMakeLists.txt

```
@@ -274,3 +274,10 @@ install(TARGETS ${INSTALL_TARGETS}
   LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}
   ARCHIVE DESTINATION ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}
 )
+
+if (USE_NEKRS)
+  install(FILES scripts/summit_bsub_openmc_nekrs.sh 
+    TYPE BIN 
+    PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
+  )
```

Using `PROGRAMS` should get the execute bit on without needing to specify `PERMISSIONS`
```suggestion
  install(PROGRAMS scripts/summit_bsub_openmc_nekrs.sh TYPE BIN)
```

---

Yep, that works!